### DISCUSSION TITLE
Server license

### DISCUSSION THREAD

**Discussion by Maniues** [toeverything/AFFiNE#4471] [General]
This project claims that it is open source, but the server code is available under a strange license.

After reading it, I think it is not open source, and AFFiNE is not as open source as if it could be. I can't use this server for my home use or anything.

I understand that this is probably part of the business model, but in that case why this project falsely states that it is open source? The application is great, but synchronization and other things in the cloud are very needed things today.

Did you not think about the AGPL license?
And so most will use the official server for which they would have to pay.

**Comment by HeJiachen-PM** [comment]
Hi. We don't think open source necessarily means free. AFFiNE is a open source project with mixed license and the app itself is completely open source under MIT license. And yes, the server logic part isn't open source.

**Comment by Brooooooklyn** [comment]
@Maniues we will separate the `apps/server` into **Community Edition**  and **Enterprise Edition** in the future; put them into different LICENSE and restrictions. For personal usage, the **Community Edition** should be free to self-host and use.

**Comment by Maniues** [comment]
Hi. Yes of course, open source doesn't mean it's free in meaning I don't pay for it.


Thanks for explaining about server community and enterprise edition.
Does this mean community edition will be licensed under open source license?

**Reply by Brooooooklyn** [reply]
> Does this mean community edition will be licensed under open source license?

Sure, we just need more time to determine which features need to be in Enterprise Edition, the rest will be under the MIT  LICENSE

**Reply by Maniues** [reply]
Thank you for explaining this. I'm waiting for this decision.
